src/Sentry.EntityFramework/Sentry.EntityFramework.csproj

```diff
@@ -1,11 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;net462;netstandard2.1</TargetFrameworks>
```

Comment:
Do we need to add this new target? This package isn't something evolving so best to leave it with the old targets and eventually move it out of this repo

Reply:
See https://github.com/getsentry/sentry-dotnet/pull/2793#discussion_r1383992723